tests/test_run_build.py

```diff
@@ -88,3 +98,65 @@ def test_main(self, mock_temp, mock_repo, mock_recorder, mock_builder, *mocks):
 
         # the output manifest is written
         mock_recorder.return_value.write_manifest.assert_called()
+
+    OPENSEARCH_DASHBOARDS_MANIFEST = os.path.realpath(
+        os.path.join(
+            os.path.dirname(__file__), "../manifests/1.1.0/opensearch-dashboards-1.1.0.yml"
```

Comment:
We haven't merged #767 yet, but start taking the habit of using `os.path.join` vs. `/`.